src/main/java/pl/tomaszdziurko/jvm_bloggers/UTMParametrsConstants.java

```diff
@@ -0,0 +1,25 @@
+package pl.tomaszdziurko.jvm_bloggers;
+
+/**
+ * UTM (Urchin Traffic Monitor) URL query parameter identifiers.
+ * 
+ * @author cslysy <jakub.sprega@gmail.com>
+ */
+public class UTMParametrsConstants {
```

Comment:
Let's keep it in a separate class as we're going to add some more links (website with archive newsletter issues is on the roadmap, also maybe Twitter/FB) so we're going to need this in other places too.